concent_api/utils/helpers.py

```diff
@@ -2,14 +2,30 @@
 import base64
 import binascii
 import datetime
+import os
 import time
 
+from django.conf                    import settings
+from django.core.checks             import Error
+from django.core.checks             import register
 from django.utils                   import timezone
 
 from golem_messages                 import message
 from golem_messages.datastructures  import FrozenDict
 
 
+@register
+def storage_cluster_certificate_path_check(app_configs = None, **kwargs):
+    errors = []
+    certificate_path = settings.STORAGE_CLUSTER_SSL_CERTIFICATE_PATH
```

Comment:
I don't see a reason for creating extra variable used only once. Above you are checking value directly from settings.